src/providers/info.rs

```diff
@@ -1,3 +1,5 @@
+use crate::cache::cache::Cache;
```

Comment:
cache cache cache :P ... would love a better naming, but not critical :P

Reply:
Totally, I would rename to `redis`  as this is the redis specfiic impl of this cache, if you agree.

Reply:
so it looks like `cache:;redis::Cache`

Reply:
but the `Cache` trait is not redis specifc. I can implement that trait also with a `SQLite`.

I do agree that the `impl Cache for ServiceCache` is redis specific.

Reply:
I would ask you then to provide the suggestion for a better name as I can't think of a better option.

Reply:
the only thing that isn't redis specific is the `Cache` trait, which I suppose I could move to `mod.rs` which would still result in `crate::cache::Cache`. If you are ok with that then I would:
- Move `trait Cache` to `mod.rs`
- Rename `cache.rs` to `redis.rs`